### ISSUE TITLE
[FR]: postfix undeliverable address

### ISSUE THREAD

**Issue by fanto666** [fail2ban/fail2ban#3039]
### Environment:

- Fail2Ban version  : 0.10.2-2.1
- OS, including release name/version : Debian GNU/Linux 10 (buster)

#### Service, project or product which log or journal should be monitored

- Name of filter or jail in Fail2Ban (if already exists) : postfix.conf
- Service, project or product name, including release name/version : postfix 3.4.14-0+deb10u1

#### Log or journal information

- Log file name(s) : /var/log/mail.log

#### Any additional information

when postfix uses sender/recipient verification, "undeliverable address" is sent for undeliverable addresses which works as "User unknown". 
This message is hardcoded, not changeable via admin (admin can change rest of message tho).

adding "undeliverable address" to mdpr-normal fixes this and catches the lines:

```
mdre-normal=^%(_pref)s from [^[]*\[<HOST>\]%(_port)s: [45][50][04] [45]\.\d\.\d+ (?:(?:<[^>]*>)?: )?(?:(?:Helo command|(?:Sender|Recipient) address) rejected: )?(?:Service unavailable|User unknown|(?:Client host|Command|Data command) rejected|Relay access denied|(?:Host|Domain) not found|need fully-qualified hostname|match|undeliverable address)\b
```

#### failures in sense of fail2ban filter (fail2ban must match):

```
May 19 00:00:30 proxy2 postfix/smtpd[16123]: NOQUEUE: reject: RCPT from example.net[192.0.2.1]: 550 5.1.1 <riaditel@example.com>: Recipient address rejected: undeliverable address: verification failed; from=<earlgarden7008@example.org> to=<riaditel@example.com> proto=ESMTP helo=<example.net>
```

**Comment by sebres** [comment]
Well, since c5f1598a215b337ce5a93243fe17896edff780e6 `Recipient address rejected` will be already recognized with any code, even with `550 5.1.1` but, as you correctly wrote, not with `undeliverable address: verification failed`.

Just I'm not sure it should be considered as a failure in sense of fail2ban **by default**, because it may affect legitimate users mistakenly or inadvertently used wrong address in TO or CC/BCC.
Sure, the same may be valid for "User unknown", but it is historically there (and still remained for backwards compatibility reasons)...
Normally both `User unknown` and `undeliverable address` would belong to the mode `aggressive` in my opinion, but unlike all other filters this has modes `auth`/`auth2`, thus I would say let's do it so.

**Comment by sebres** [comment]
Implemented in 43f2923fbd2507b6aa9c357ead261f1800555ec6.
